agents/labjack/labjack_agent.py

```diff
@@ -270,8 +271,14 @@ def start_acq(self, session, params=None):
 
             # Start the data stream. Use the scan rate returned by the stream,
             # which should be the same as the input scan rate.
-            scan_rate = ljm.eStreamStart(self.handle, scans_per_read, num_chs,
-                                         ch_addrs, scan_rate_input)
+            try:
+                scan_rate = ljm.eStreamStart(self.handle, scans_per_read, num_chs,
+                                             ch_addrs, scan_rate_input)
+            except LJMError: #in case the stream is running
```

Comment:
There are quite a few different [error codes](https://github.com/labjack/labjack-ljm-python/blob/master/labjack/ljm/errorcodes.py) associated with LJMError. Can we log more info about the type of error raised, i.e.:
```
except LJMError as e:
    print(e)
    print("Stopping and restarting stream.")
```
The `print(e)` will print the associated errorString, i.e. `LJM library error code 1227 LJME_DEVICE_NOT_FOUND`.